### ISSUE TITLE
Trina's old request 1

### ISSUE THREAD

**Issue by td-spec** [trinzz/trinzz.github.io#39]
## Goal
Rewrite and redesign the homepage to position Trinzz as:
**the clinical-grade data infrastructure layer for diagnostic AI and precision medicine** (Scale/NVIDIA authority).
The UI must feel modern, premium, and “AI-infrastructure”:
- predominantly white background
- blue sections + blue→purple gradient highlights
- subtle animated background elements (non-distracting)
Primary conversion goal: **Book a demo / Speak to an Expert**
Secondary: **Explore Solutions**

## Current Problems to Fix
- Homepage currently reads like a generic “training data platform” and includes “labeling services” framing. :contentReference[oaicite:1]{index=1}
- We must elevate positioning to “infrastructure layer” and reduce vendor/outsourcing perception.

---

## Route
- `/` (homepage)

---

## Target Audience (Homepage ICP)
- Series A–C diagnostic AI startups (CTO/Head of ML)
- Hospital innovation + radiology/pathology leadership
- Diagnostic networks and labs
- Medical device / imaging software teams

Homepage should speak to engineers + clinical leaders:
- reliability under distribution shift
- traceability and governance
- deployment readiness and validation
- imaging-native workflows (DICOM, NIfTI, WSI, 3D)

---

### Theme
- Background: white
- Accents: #1E6BFF (blue) + blue→purple gradient (e.g., #1E6BFF → #7A3FF2)
- Use “blue section bands” for emphasis (1–2 full-width sections)

### Subtle AI Motion (must be tasteful)
Implement these animation layers (choose 2–3 max, not all):
1) **Gradient glow blob** behind hero headline (slow, low opacity, blur)
2) **Dot-grid parallax** (tiny dots, slow drift, fades near edges)
3) **Animated network lines** (SVG paths with very slow dash offset)
4) **Scroll reveal** for sections (Framer Motion: fade + translateY 8–16px)
5) **Micro-interactions**: hover lift on cards, CTA glow on hover

Constraints:
- No heavy particle animations
- Must respect `prefers-reduced-motion`
- Keep CPU/GPU light

### Our stack and components can include
- Framer Motion (preferred)
- Optional: lightweight SVG background components (no canvas needed)

---

## Homepage Information Architecture (Sections)
1) Top Nav (Solutions, Product, Security, Company, Contact) + CTA button
2) Hero (infrastructure positioning) + 2 CTAs
3) “Why deployments fail” (data infra gap)
4) “What Trinzz is” (infrastructure layer) + 4 pillars
5) Solutions tiles (3 groups) with “Learn more” buttons
6) Product/Platform preview (capabilities)
7) Trust + Security (SOC2/HIPAA/GDPR phrased carefully; only include if true in your system)
8) How it works (4 steps)
9) Final CTA band

---

### 1) Hero
**Eyebrow:** Clinical-Grade Data Infrastructure  
**H1:** The Infrastructure Layer That Makes Medical AI Deployable  
**Subhead:** Most diagnostic AI breaks in real clinical settings—not because of models, but because data pipelines are not built for reliability, traceability, and continuous validation. Trinzz turns complex clinical imaging and multimodal data into production-ready datasets engineered for deployment and institutional trust.  
**Format line:** Native support for DICOM • NIfTI • WSI • 3D/volumetric  
**Primary CTA:** Book a demo  
**Secondary CTA:** Explore Solutions

# GitHub Copilot Add-On: Add “Trust Strip” Under Hero (SOC 2 Type II + HIPAA + ISO 27001)

## Goal
Add a compact “Trust strip” directly under the homepage hero so enterprise buyers see compliance within 3 seconds. It should reinforce trust without clutter and match the white + blue + blue-purple highlight theme.

Badges must display:
- SOC 2 Type II
- HIPAA
- ISO 27001

---

## Placement
Homepage (`/`):
Insert immediately below the Hero CTAs (Book a demo / Explore Solutions), before the next section.

Component name:
`TrustStrip`

---

## UI / Design Requirements
- Background: white
- Height: compact (e.g., 44–56px)
- Border: subtle 1px border or hairline divider (light gray) OR a faint blue→purple top accent line
- Layout (desktop):
  - Left: small text label
  - Right: three badges inline
- Layout (mobile):
  - stack or horizontal scroll (no wrapping into 2 messy lines)
- Visual style:
  - badges are small pills with subtle border
  - optional minimal shield/check icon
  - use blue-purple accent on icon or badge edge only (keep understated)

---

## Copy (Use Exactly)
Left label text:
**Certified for clinical-grade security:**

Badges:
**SOC 2 Type II** | **HIPAA** | **ISO 27001**

Optional microtext (small, muted) under label on desktop only:
**Enterprise-ready governance and auditability for clinical data.**

---

## Interaction
- Badges are clickable and route to `/security` if it exists.
- If `/security` does not exist, route to `/#security-compliance` (anchor to the Security & Compliance section).
- Hover: subtle background tint
- Focus: visible outline for accessibility

---

## Accessibility
- Use semantic structure:
  - `<section aria-label="Security certifications">`
- Badge links must be keyboard navigable.
- Ensure color contrast meets WCAG.

---

## Implementation Notes
Create:
- `components/home/TrustStrip.tsx`
- Reuse existing `Badge` component if available; otherwise create `components/ui/Badge.tsx`.

Add an id to the Security section for anchor navigation:
- `<section id="security-compliance">`

---

## Reduced Motion
No animation required in trust strip.
If you add any hover transitions, keep them minimal and respect `prefers-reduced-motion`.

---

## Acceptance Criteria
- Trust strip appears under hero on all screen sizes
- Shows: SOC 2 Type II, HIPAA, ISO 27001 within first viewport
- Click routes to `/security` or `#security-compliance`
- Looks clean, premium, and not cluttered
- Keyboard accessible with visible focus states

**Trust chips (small):**
- Reliability metrics
- Dataset lineage
- Expert-in-the-loop validation
- Deployment readiness

---

### 2) “Why deployments fail”
**H2:** Why Medical AI Fails After the Benchmark  
**Body:** Validation accuracy is not deployment reliability. Real-world environments introduce scanner variability, protocol drift, edge cases, and institutional differences. If your dataset lacks governance, measurable reliability, and traceable provenance, performance degrades and trust collapses.  
**Bullets:**
- Distribution shift across sites, scanners, and populations
- Edge cases missing from training and evaluation datasets
- Label inconsistency without expert consensus workflows
- No dataset lineage for audit, validation, or regulatory evidence

---

### 3) “What Trinzz is”
**H2:** Trinzz Is Not Labeling. It’s Infrastructure.  
**Body:** Trinzz provides the data infrastructure layer for diagnostic AI: expert-validated dataset construction, measurable reliability, and end-to-end provenance—so teams can deploy with confidence and scale across institutions.  
**4 pillar cards:**
1) **Reliability, Quantified** — consensus workflows + measurable quality gates  
2) **Lineage, Built-In** — every label, revision, and validation step traceable  
3) **Imaging-Native Workflows** — DICOM/WSI/3D viewers and clinical QA flows  
4) **Deployment-Ready Datasets** — engineered for institutional validation and real-world performance

---

### 4) Solutions Tiles (with Learn more)
**H2:** Built for the Teams Shipping Clinical AI  
**Subhead:** Choose your deployment environment. Trinzz infrastructure adapts to your buyer reality.

Render 3 grouped blocks with tiles and “Learn more” buttons:

#### Group A: Diagnostics AI Infrastructure
- **Diagnostic AI Startups**  
  Text: Move from pilots to production with deployment-grade datasets, edge-case loops, and traceable lineage.  
  Button: Learn more → `/solutions/diagnostic-ai-startups`

- **National Diagnostic Networks**  
  Text: Standardize reliability across distributed sites with governance, QA, and continuous validation.  
  Button: Learn more → `/solutions/national-diagnostic-networks`

- **Enterprise Hospital Labs**  
  Text: Deploy AI safely inside radiology and pathology workflows with clinical-grade validation infrastructure.  
  Button: Learn more → `/solutions/enterprise-hospital-labs`

- **Independent Diagnostic Labs**  
  Text: Improve consistency and throughput with AI-ready datasets and operational QC.  
  Button: Learn more → `/solutions/independent-diagnostic-labs`

Add a small “Learn more about Diagnostics AI Infrastructure” link → `/solutions/diagnostics-data-infrastructure`

#### Group B: Clinical Imaging Infrastructure
- **Radiology AI** → `/solutions/radiology-ai`
- **Pathology Imaging** → `/solutions/pathology-imaging`
- **Volumetric Imaging** → `/solutions/volumetric-imaging`
(Each tile includes 1-line microcopy; keep concise)
Add “Learn more” → `/solutions/clinical-imaging-infrastructure`

#### Group C: Life Sciences Infrastructure
- **Oncology Data Infrastructure** → `/solutions/oncology-data-infrastructure`
- **Multiomics & Drug Discovery Infrastructure** → `/solutions/multiomics-drug-discovery-infrastructure`
Add “Learn more” → `/solutions/life-sciences-infrastructure`

---

### 5) Platform Capabilities (Product preview)
**H2:** Everything You Need to Build Trusted Datasets  
**Cards (6):**
- Imaging-native viewers (DICOM, NIfTI, WSI, 3D)
- Expert-in-the-loop workflows (review, adjudication, consensus)
- Quality gates & reliability scoring
- Dataset versioning and lineage
- Edge-case mining & failure set creation
- API/SDK for integration

Add small CTA: **Book A Demo**.

---

### 6) How it works
**H2:** From Raw Clinical Data to Deployment-Ready Datasets  
**Steps:**
1) Ingest clinical imaging and metadata (secure, governed access)
2) Build ground truth with expert validation + reliability gates
3) Create evaluation sets that reflect real deployment conditions
4) Version, trace, and deliver datasets for training, QA, and submission evidence

---

## Goal
Add a high-trust “Security & Compliance” section to the homepage that immediately clears enterprise security objections and increases conversion for hospitals, diagnostic chains, and Series A–C AI companies.

We have:
- SOC 2 Type II
- HIPAA
- ISO 27001

These must be displayed as real compliance badges and reinforced with concrete security controls.

---

## Section Placement
Homepage: place after “Platform Capabilities” and before “How it works”.
Component name: `SecurityComplianceSection`

---

## Visual Requirements (Premium, cool, subtle)
- Background: white
- Add a thin gradient accent bar at top (blue → purple)
- Add an ultra-subtle animated “shield grid” or “dot network” SVG behind the section (low opacity, slow movement)
- Use 2-row layout:
  - Row 1: headline + description + compliance badges
  - Row 2: 4 security cards + “Request security pack” CTA panel

Animation:
- Use Framer Motion for fade-up on scroll
- Respect `prefers-reduced-motion` (disable background animation + reduce transitions)

---

### Eyebrow
Security & Compliance

### Headline
Enterprise-Grade Security Built for Clinical Data

### Body
Clinical data requires more than “secure enough.” Trinzz is built for healthcare environments where privacy, access control, and auditability are non-negotiable. Our infrastructure is certified and designed to keep sensitive imaging workflows secure—while maintaining dataset traceability institutions require.

---

## Compliance Badges (Must Show)
Render badges prominently (top right or directly under headline):

- SOC 2 Type II
- HIPAA
- ISO 27001

Badge style:
- pill or small card badges
- include a small shield icon (optional)
- no long paragraphs—keep clean

---

## Security Pillars (4 Cards)
Render 4 cards with titles + copy:

1) **Access Control & Least Privilege**
Role-based access control and permissioned workflows designed for clinical teams and external reviewers.

2) **Auditability & Dataset Traceability**
Audit logs and end-to-end provenance across dataset actions, including annotation, review, validation, and export.

3) **Encryption & Secure Transport**
Encryption in transit and at rest, with secure access patterns and hardened endpoints for clinical workflows.

4) **Enterprise Governance**
Controls designed for enterprise risk reviews, including environment separation patterns and operational governance best practices.

---

## “Security Pack” CTA Panel (Must)
Add a CTA panel to the right of the cards (or below on mobile):

### Title
Need a security review?

### Body
Request our enterprise security pack for due diligence: SOC 2 Type II report availability, ISO 27001 certificate, HIPAA documentation, security architecture overview, and standard security controls summary.

### Button (Primary)
Request security pack

### Secondary link
Contact security

Implementation:
- Button routes to `/contact?reason=security-pack` (or existing contact route)
- Secondary link: `mailto:security@trinzz.com` (ONLY if this mailbox exists; otherwise route to contact)

---

## Optional “Data Handling” Callout (Use If Accurate)
If Trinzz supports customer-controlled storage / customer-managed buckets, show this callout.
If not, omit entirely.

Callout title:
Data access designed for clinical governance

Callout body:
Data access is governed through scoped permissions, audit logs, and controlled workflows designed for regulated environments.

---

## Technical Requirements
Create:
- `components/home/SecurityComplianceSection.tsx`
- `components/ui/Badge.tsx` (if not present)
- `config/securityClaims.ts` (optional, but can hardcode since claims are true)

If using config, set:
```ts
export const securityClaims = {
  soc2Type2: true,
  hipaa: true,
  iso27001: true
};

---

### 8) Final CTA band (Blue section)
**H2:** Build Medical AI That Holds Up in the Real World  
**Body:** Replace fragile data pipelines with clinical-grade infrastructure built for reliability, traceability, and deployment.  
Buttons: **Request pilot** (primary) + **Book a demo** (secondary)

---

## UI Layout Notes
- Use large whitespace and strong type scale.
- Use 1–2 blue background bands (e.g., Final CTA + one mid “Infrastructure” band).
- Use gradient highlight on key words (e.g., “Infrastructure Layer”, “Deployable”).
- Use subtle animated background behind hero only (do not animate every section).

---

## Implementation Tasks
1) Create `pages/index.tsx` (or app router equivalent)
2) Build reusable components:
   - `HeroSection`
   - `SectionHeader`
   - `PillarsGrid`
   - `SolutionsMegaTiles`
   - `CapabilitiesGrid`
   - `HowItWorks`
   - `TrustSecurity`
   - `CTABand`
   - `AnimatedBackdrop` (SVG + Framer Motion)
3) Add `prefers-reduced-motion` handling:
   - If reduced motion, disable background animation and use static gradients.

---

## Acceptance Criteria
- Homepage feels premium, “infrastructure-first,” and not like an annotation marketplace.
- Clear narrative: deployment failures → infrastructure → Trinzz.
- Subtle AI motion in hero background (tasteful, low CPU).
- Solutions tiles include **Learn more** buttons routing correctly.
- Mobile responsive + accessible.
- CTA buttons visible in hero + bottom band.
